### ISSUE TITLE
Using SwiftyBeaver from multiple targets

### ISSUE THREAD

**Issue by allewun** [SwiftyBeaver/SwiftyBeaver#193]
My iOS app has a main app target "`MyApp`" and a framework target "`MyFramework`" linked into the main app target.

I'd like to be able to use SwiftyBeaver from both targets. Here's what I did:

**Podfile:**
```ruby
target 'MyApp'
end

target 'MyFramework'
  pod 'SwiftyBeaver'
end
```

**MyFramework : Logger.swift**
```swift
import SwiftyBeaver

let log = Logger(module: "MyFramework")

public struct Logger {
    public init(module: String = "") {
        let console = ConsoleDestination()
        console.format = "\(module) $N.$F:$l - $M"
        SwiftyBeaver.addDestination(console)
    }

    // methods to relay calls to SwiftyBeaver
    public func verbose( ... ) 
    ...
}
```

**MyApp : AppDelegate.swift**

```swift
import MyFramework

let log = Logger()

class AppDelegate {
    // ...
}
```

The problem I'm running into is once each of the `log` singletons (one in the main target and one in the framework) has been initialized for the first time, all subsequent console log events happen twice.

I dug into the source and this seems by design. When you call `SwiftyBeaver.addDestination()`, the 2nd `ConsoleDestination()` will be added to the `destinations` set and all future logs appear to be duplicated, since events are dispatched to all `destinations`.

I don't currently have any ideas how to amend the `destinations` set to support my usecase. I'm not sure how to cleanly identify which `ConsoleDestination()` I should be referencing. 

Is there a better way to achieve what I'm trying to do? Please let me know if I should clarify anything. There's a lot going on here 😄 

**tl;dr:**
I'd like to be able to do `log.debug(...)` from both `MyApp` and `MyFramework` targets. Bonus points if I can customize the `.format` for each, so that I can distinguish between them in the console. My current setup leads to duplicated logs.

**Comment by skreutzberger** [comment]
That is currently not possible. You need to define individual logger objects (one in each target).

**Comment by allewun** [comment]
Follow-up question: is it intended that `ConsoleDestination() == ConsoleDestination()` ?

From the equality implementation, we have:

```swift
public func == (lhs: BaseDestination, rhs: BaseDestination) -> Bool {
    return ObjectIdentifier(lhs) == ObjectIdentifier(rhs)
}
```

Identically instantiated `destinations` are treated as distinct. Would it make more sense for them to be equal?
